src/ml_tooling/data/sql.py

```diff
@@ -0,0 +1,28 @@
+import abc
+from typing import Tuple
+
+from ml_tooling.data.base_data import DataSet
+from ml_tooling.utils import DataType
+import sqlalchemy as sa
+from sqlalchemy.engine import Connectable
+
+
+class SQLDataSet(DataSet, metaclass=abc.ABCMeta):
```

Comment:
I would be convenient to have dump method too. That way the data could be dumped to another DB accessibile in porduction.

Reply:
I think we need some more experience with how that would look - we can do some adhocs when creating our subclass in our current APIs and add it in to ML tooling when we have something that works?